deploy/manifests/BUILD.bazel

```diff
@@ -13,12 +13,14 @@ RELEASE_NAMESPACE = "cert-manager"
 VARIANTS = {
     "cert-manager": {
         "crd_variant": "regular",
+        "additional_api_versions": "",
         "values": {
             "installCRDs": "false",
         },
     },
     "cert-manager-legacy": {
         "crd_variant": "legacy",
+        "additional_api_versions": "hacking-helm.i-wish-this-wasnt-required.cert-manager.io/use-v1beta1-webhooks",
```

Comment:
It's not clear to me why we are doing this. I guess it's a way to force v1beta1 in legacy? If so maybe call it `force-v1beta1-webhooks`?

Reply:
we should document this in detail a bit more exactly... there is a whole story behind this. Will add more comments

Reply:
Your suggestion made sense so changed it as well as adding an extra comment to the bazel file where this weird name comes from

Reply:
So because it uses the API versions for the k8s version that Helm was compiled with it the naive check will cause it to use v1 or not depending on which version of helm is used, and this allows us to force v1beta1 for legacy even when using a newer Helm?

Reply:
Yup!